prov/usnic/src/usdf_cm.c

```diff
@@ -235,13 +238,32 @@ usdf_cm_msg_connreq_failed(struct usdf_connreq *crp, int error)
 		eq = pep->pep_eq;
 	}
 
+	reqp = (struct usdf_connreq_msg *) crp->cr_data;
```

Comment:
On second thought maybe this whole part should be a separate function since this form of failure is a special case.